### PR TITLE
feat(docs): add ```Accordion``` theming docs

### LINKED ISSUE #1035: Theming: Accordion
### Subject

Accordion

### Description

Create a basic theming documentation for the `Accordion` component that showcases how you can style it.

Theming file: [Accordion theming file](https://github.com/chakra-ui/chakra-ui-docs/blob/main/content/docs/components/accordion/theming.mdx)

### Template Reference

For single part components, feel free to use the `Button` component theming doc as a template reference: [Button theming](https://github.com/chakra-ui/chakra-ui-docs/blob/main/content/docs/components/button/theming.mdx).

For multipart components, feel free to use the `Input` component theming doc as a template reference: [Input theming](https://github.com/chakra-ui/chakra-ui-docs/blob/main/content/docs/components/input/theming.mdx).

### PR DESCRIPTION
**Pull request by HariVP03**
Closes #1035 

## 📝 Description
Adds the theming docs for ```Accordion``` component.

## ⛳️ Current behavior (updates)
There are no theming docs for ```Accordion``` currently

## 🚀 New behavior
Adds the theming docs for ```Accordion``` component.

## 💣 Is this a breaking change (Yes/No):
No

### CODE REVIEW

**Review by nikolovlazar — CHANGES_REQUESTED** [review]
Thanks for opening this PR, @HariVP03. But I can see you're not following the template I've provided in the GitHub Issue #1035. Could you refactor your PR to reflect on that?

**Comment by HariVP03** [comment]
> 

Sure

**Comment by HariVP03** [comment]
@nikolovlazar I've made the necessary changes to follow the new format

File: content/docs/components/accordion/theming.mdx [hunk 1]
```diff
@@ -2,3 +2,188 @@
...
 scope: theming
 ---
+
+The `Accordion` component is a multipart component. The styling needs to be
+applied to each part specifically.
+
+> To learn more about styling multipart components, visit the
+> [Component Style](/docs/styled-system/component-style#styling-multipart-components)
+> page.
+
```
Comment (nikolovlazar) [inline content/docs/components/accordion/theming.mdx:12]: I've prepared an anatomy illustration for you. Grab [this code](https://gist.github.com/nikolovlazar/bec580e8d8d2eb12bed7c7aa6086275a) and add the following section as the first:

```markdown
## Anatomy

- A: `root`
- B: `container`
- C: `button`
- D: `panel`
- E: `icon`

[PLACE THE ILLUSTRATION CODE HERE]
```

**Review by nikolovlazar — CHANGES_REQUESTED** [review]
Nearly there, @HariVP03!

I noticed you've got a couple of `.ts` extensions left in the imports. I've also provided you with the accordion anatomy illustration code. Apply these changes and we're good to go!

**Review by nikolovlazar — APPROVED** [review]
All good!

**Comment by nikolovlazar** [comment]
@allcontributors please add @HariVP03 for docs.

**Pull request merged**